etc/config-template.toml

```diff
@@ -216,6 +216,11 @@
 ## 32MB during checking and set it back to the default value in normal operations.
 # region-split-check-diff = "6MB"
 
+## The threshold of triggering Region split check.
+## When Region keys number change exceeds this config, TiKV will check whether the Region should be split
+## or not.
+# region-split-check-keys-diff = "60000"
```

Comment:
em, is it a proper default value? 

/cc @zhangjinpeng1987 @BusyJay @disksing

Reply:
LGTM

Reply:
If key/value size is 100 bytes, 60,000 key/value pairs is about 6MB, but if the key/value size is 50 bytes, 60,000 key/value pairs is about 3MB, i think 60,000 is too small. How about 100,000?

Reply:
@zhangjinpeng1987  We already have region_split_check_size_diff to check size, maybe it's not necessary to consider kv size here?

Reply:
@zhangjinpeng1987 We already have region_split_check_size_diff to check size, maybe it's not necessary to consider kv size here?